parsing values outside of the short domain (#158)
KxSystems/docs: docs/ref/tok.md

2015.10.28D03:55:58.654200000
```

> ## Outside of Domain

Reviewer: Titles in sentence case pls. This is an initial capital, then all lower case UNLESS the word is a proper noun, i.e. the name of something, e.g. Jo or Microsoft.

```suggestion
## Outside of domain
```

> Parsing values outside of the types domain will return null

Reviewer: Pls try to keep phrasing in the present tense:

```suggestion
Parsing values outside of the types domain returns null.
```

```q
q)"H"$"32768"
0Nh
q)"I"$"2147483648"
0Ni
q)"D"$"2147483648"
0Nd
```

> !!! note short converting to 0Nh instead of +/-0Wh since 4.1t 2021.09.03,4.0 2021.10.01

Reviewer: Does this make sense? (It doesn't to me, but...) 
And should the "s" be a capital?

```suggestion
!!! note Short converting to 0Nh instead of +/-0Wh since 4.1t 2021.09.03,4.0 2021.10.01
```
